test/test_tensorexpr.py

```diff
@@ -1595,6 +1595,26 @@ def getModule(script):
 
         torch.testing.assert_allclose(ref, test)
 
+    @unittest.skipIf(not torch.cuda.is_available(), "requires CUDA")
+    def test_multiple_outputs(self):
+        # A bug reported internally similar to the one reported in #48533
+        def foo(a, b, c):
+            t_next = c + 1
+            t5 = t_next * b
+            t6 = torch.unsqueeze(t_next, 1)
+            t7 = a * t6
+            return (t7, t5, t_next)
+
+        a = torch.rand(20, 20, dtype=torch.float32, device='cuda')
+        b = torch.rand(20 * 29, dtype=torch.float32, device='cuda').as_strided([20], [29])
+        c = torch.ones(20, dtype=torch.int64, device='cuda')
+        traced = torch.jit.trace(foo, (a, b, c))
+        ref = foo(a, b, c)
+        exp = traced(a, b, c)
+        exp = traced(a, b, c)
```

Comment:
Hello,
   I was wondering if line 1614 is intentionally added in the test?  ( line 1613 & 1614 are identical ) 
   Since foo is not using any 'rand' so, calling traced on same input values a,b &c should result in the same exp tensor being calculated twice.

Reply:
Yes, it is intentional. We call it twice because the first run is used for profiling and only on the second run the fuser kicks in.